frontend/src/metabase/static-viz/lib/numbers.unit.spec.js

```diff
@@ -46,7 +46,7 @@ describe("formatNumber", () => {
       number_style: "scientific",
     });
 
-    expect(text).toEqual("1.2E3");
+    expect(text).toEqual("1.2e+3");
```

Comment:
this is how our main app format scientific number, so it makes sense to adjust test to the new behavior.